app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for i in people:
+        for key in i:
+            Person(i["name"], i["age"])
+    for j in people:
```

Comment:
Please, change the name of this variable to `person` or something more informative